src/LabelsProviderInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator;
+
+/**
+ * Provides data attribute labels.
+ */
+interface LabelsProviderInterface
+{
+    /**
+     * @return array<string, string> A set of attribute labels.
+     */
+    public function getLabels(): array;
```

Comment:
```suggestion
    public function getValidationPropertyLabels(): array;
```

`getLabels` it's more common name and may lead to conflicts in some cases.